projects/rubygems.org/package.yml

```diff
@@ -24,12 +24,12 @@ build:
         --no-rdoc
 
     - run: |
-        patch -p1 < "$SRCROOT"/props/fit-n-finish.patch
+        patch -p1 < "$SRCROOT"/props/fit-n-finish.1.patch
         patch -p1 < "$SRCROOT"/props/fit-n-finish.2.patch
-        sed -i -e 's/Gem.default_dir/Gem.default_system_dir/' lib/rubygems/path_support.rb
+        sed -ie 's/Gem.default_dir/Gem.default_system_dir/' lib/rubygems/path_support.rb
```

Comment:
did you check this? i think this makes a backup with `e` on the end:
```sh
$ echo hi >foo
$ pkgx +sed sed -ie 's/hi/bye/' foo
$ cat foo
bye
$ ls
foo	fooe
```

Reply:
Oops if true.